lighthouse/models.py

```diff
@@ -165,35 +165,35 @@ def predict(self, query):
 
 
 class MomentDETRPredictor(BasePredictor):
-    def __init__(self, ckpt_path, device, feature_name, slowfast_path, pann_path):
+    def __init__(self, ckpt_path, device, feature_name, slowfast_path, pann_path=None):
```

Comment:
Is "pann_path=None" necessary? I think that the BasePredictor has already set pann_path=None as a default value. Without pann_path=None, I guess that it does not have any errors.

Reply:
I think that MomentDETRPredictor is called before BasePredictor and BasePredictor does not necessarily need pann_path, bad MomentDETRPredictor definitely need it. Therefore this changes is good for preventing issues related to pann_path.